prog/weaveutil/image.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	docker "github.com/fsouza/go-dockerclient"
+)
+
+func pullImage(args []string) error {
```

Comment:
@bboreham What's your thoughts on me removing this command (and file) and following your suggestion here: https://github.com/weaveworks/weave/issues/2957#issuecomment-327787115

Reply:
I went ahead and did it, since it was a simple change and I can back out if you decide you want the separate `weaveutil` command.